builtins.c

```diff
@@ -126,10 +128,10 @@ int	ft_strisnumeric(char *str)
 	size_t	i;
 
 	i = 0;
-	while (str[i] && (ft_isspace(str[i]) || str[i] == '+'))
+	while (str[i] && (ft_isspace(str[i]) || str[i] == '+' || str[i] == '-'))
```

Comment:
We should change this into an if statement of split it into

	while (str[i] && ft_isspace(str[i]) 
	    i++;
	if (str[i] == '+' || str[i] == '-')
	    i++;

Reply:
oh wow - thing is if there are more minuses they would change into positivity - so we should also take this into account?